sklearn/cluster/k_means_.py

```diff
@@ -1423,8 +1384,8 @@ class MiniBatchKMeans(KMeans):
     >>> kmeans = kmeans.partial_fit(X[0:6,:])
     >>> kmeans = kmeans.partial_fit(X[6:12,:])
     >>> kmeans.cluster_centers_
-    array([[1, 1],
-           [3, 4]])
+    array([[2. , 1. ],
```

Comment:
Is this a bugfix? Should there be a *changed estimators* entry in the what's new?

Reply:
I haven't filled the what's new yet, but there are several bug fixes included in this PR. I don't know yet how to display them in the what's new